libraries/AP_HAL_ChibiOS/hwdef/include/minimize_fpv_osd.inc

```diff
@@ -0,0 +1 @@
+include "minimize_common.inc"
```

Comment:
Maybe we should put a comment at the top of each of these inc files clarifying which boards it applies to and the hierarchy with other inc.